site/content/docs/traces-open-telemetry/importing-traces/sending-traces-otel-collector.md

```diff
@@ -1,23 +1,28 @@
 ---
-title: Sending traces to Checkly with the OpenTelemetry collector
+title: Setting up the OpenTelemetry collector
 
 metatags:
-  title: "Sending traces to Checkly with the OpenTelemetry collector"
-  description: "Send traces to Checkly using the OpenTelemetry collector."
+  title: "Setting up the OpenTelemetry collector"
+  description: "Set up the OpenTelemetry collector and send traces to Checkly."
 weight: 20
 menu:
-  integrations:
-    parent: "OpenTelemetry (beta)"
+  platform:
+    parent: "Send your traces to Checkly"
 beta: true
 aliases:
   - "/docs/open-telemetry/ingesting-traces-otel-collector/"
+  - "/docs/open-telemetry/sending-traces-otel-collector/"
 ---
 
 Using the OpenTelemetry collector is the best way to manage and route telemetry data to Checkly. It requires
 a little setting up, but it's worth it.
 <!--more-->
-In this guide we assume you already have an OTel collector running and configured. If you don't, please reference [the
-Open Telemetry Collector getting started documentation](https://opentelemetry.io/docs/collector/installation/)
+
+## Pre-requisites
```

Comment:
I think this is commonly spelled without a '-'

Reply:
Thanks- I've been doing it with the dash for readibility but I can totally see that we've been saying Prerequisites in other pages like k8s. Changing it for consistency.
```suggestion
## Prerequisites
```